benchmarks/src/main/java/com/google/samples/apps/nowinandroid/GeneralActions.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid
+
+import android.Manifest.permission
+import android.os.Build.VERSION.SDK_INT
+import android.os.Build.VERSION_CODES.TIRAMISU
+import androidx.benchmark.macro.MacrobenchmarkScope
+
+fun MacrobenchmarkScope.allowNotifications() {
+    /**
+     * Because the app under test is different from the one running the instrumentation test,
+     * the permission has to be granted manually by either tapping the Allow button
+     *
+     val obj = By.text("Allow")
+     val dialog = device.wait(Until.findObject(obj), TIMEOUT)
+     dialog?.let {
+     it.click()
+     device.wait(Until.gone(obj), 5_000)
+     }
+     *
+     * or (preferred) executing the grant command on the target package.
+     */
+    if (SDK_INT >= TIRAMISU) {
+        val command = "pm grant $packageName ${permission.POST_NOTIFICATIONS}"
+        device.executeShellCommand(command)
+    }
+}
```

Comment:
Let's make it an actual javadoc comment (with code highlighting) 😁

```suggestion
/**
 * Because the app under test is different from the one running the instrumentation test,
 * the permission has to be granted manually by either
 * 
 * - tapping the Allow button
 *    ```kotlin
 *    val obj = By.text("Allow")
 *    val dialog = device.wait(Until.findObject(obj), TIMEOUT)
 *    dialog?.let {
 *        it.click()
 *        device.wait(Until.gone(obj), 5_000)
 *    }
 *    ```
 * - or (preferred) executing the grant command on the target package.
 */
fun MacrobenchmarkScope.allowNotifications() {
    if (SDK_INT >= TIRAMISU) {
        val command = "pm grant $packageName ${permission.POST_NOTIFICATIONS}"
        device.executeShellCommand(command)
    }
}
```

Reply:
![image](https://github.com/android/nowinandroid/assets/1921278/ca03020d-b108-482a-9a5a-431db248b67f)

Reply:
Ah great idea, thanks :)